src/chrome/komodo/content/statusbar.p.js

```diff
@@ -388,14 +386,9 @@ this.setupEncodingMenu = function(menupopup)
         return;
     }
     
-    let uid = view.uid.number;
-    
-    if (_encodingMenuInitializedInViews[uid] === undefined) {
-        _encodingMenuInitializedInViews[uid] = false;
-    }
     var xv = getXulView(view);
     
-    if (!_encodingMenuInitializedInViews[uid]) {
+    if (!view._encodingWidget) {
```

Comment:
Should be `if( ! ("_encodingWidget" in view))`, otherwise it throws a warning on strict.